test/test_utils/__init__.py

```diff
@@ -1595,6 +1595,11 @@ def get_framework_and_version_from_tag(image_uri):
             "2.10.1": "2.10.1.2.7.3.0",
         },
     },
+    "2.10.0": {
```

Comment:
Can you find a better way to manage these? this was added as a temporary fix. Not a good way to handle things statically.